handlers/api-handler.ts

```diff
@@ -38,14 +38,23 @@ export interface ApiHandlerDefinition<B = never, Q = never, R = never>
 	scopes?: string[];
 	/**
 	 * Optional override of default AJV instance
+	 *
+	 * @deprecated Use your own AJV instance in `validators`
 	 */
 	ajv?: Ajv;
 
-	schemas: {
+	/** @deprecated Use `validators` instead */
+	schemas?: {
 		body?: JSONSchemaType<B>;
 		query?: JSONSchemaType<Q>;
 		response?: JSONSchemaType<R>;
 	};
+
+	validators?: {
```

Comment:
Nice--thanks for making this optional. We have a few routes that don't require schemas--and passing in an empty object felt werid.